frontend/src/topic/ApiDocs/views/index.js

```diff
@@ -0,0 +1,197 @@
+import CSSModules from 'react-css-modules';
+import Helmet from 'react-helmet';
+import React from 'react';
+
+import styles from './styles.scss';
+import { pageTitles } from '../../../common/utils/labels';
+import { RestBuilder } from '../../../public/utils/rest';
+import {
+    urlForApiDocs,
+} from '../../../common/rest';
+
+
+// Todo move this to common utils module
+const isObjectEmpty = obj => (
+    Object.keys(obj).length === 0 && obj.constructor === Object
+);
```

Comment:
I think there already is a utils functions in common.js

Reply:
I couldn't find any. Will appreciate if you could share where exactly it is implemented.